Carhartt-Work-in-Progress/sanity-plugin-media
# CAR-234 Create fork and add custom fields to asset browser edit dialogue

## Description
create a fork of the sanity media plugin to enhance the asset fields
## Changes
- [ ] add custom asset fields
- [ ] add mass edit support

## Comments for testing
steps to test locally are 
- run `git checkout -t origin/CAR-182-Create-Editorial-Image-Model` in the [sanity-b2c-repo](https://github.com/Carhartt-Work-in-Progress/b2c-sanity)
- run `rm -rf node_modules yarn.lock && yarn dev && yarn`
- navigate to the app on the port
- navigate to the asset browser 
- select multiple assets and then select edit to test the mass edit functionality
- edit a single asset to see that the following fields are present
    - Name 
    - Season (Dropdown) (Default is the current season) (Required)
    - Collaboration (Dropdown) (Optional)
    - Product(s) (Required)
    - Import Exif Data (Date, Location, etc)
    - Alt tag
